ckanext/versioning/logic/action.py

```diff
@@ -27,62 +26,38 @@ def dataset_version_update(context, data_dict):
 
```

Comment:
Should we also rename our actions so this one is called `dataset_tag_update`? Or is this a different PR?

Reply:
Done!